runtime/doc/options.txt

```diff
@@ -6204,25 +6203,13 @@ A jump table for the options with a short description can be found at |Q_op|.
 	When 'swapfile' is reset, the swap file for the current buffer is
 	immediately deleted.  When 'swapfile' is set, and 'updatecount' is
 	non-zero, a swap file is immediately created.
-	Also see |swap-file| and |'swapsync'|.
+	Also see |swap-file|.
 	If you want to open a new buffer without creating a swap file for it,
 	use the |:noswapfile| modifier.
 
 	This option is used together with 'bufhidden' and 'buftype' to
 	specify special kinds of buffers.   See |special-buffers|.
 
-						*'swapsync'* *'sws'*
-'swapsync' 'sws'	string	(default "fsync")
-			global
-	When this option is not empty a swap file is synced to disk after
-	writing to it.  This takes some time, especially on busy Unix systems.
-	When this option is empty parts of the swap file may be in memory and
-	not written to disk.  When the system crashes you may lose more work.
-	On Unix the system does a sync now and then without Vim asking for it,
-	so the disadvantage of setting this option off is small.  On some
-	systems the swap file will not be written at all. 
-	The 'fsync' option is used for the actual file.
-
```

Comment:
Should probably keep a note here given what's been done with the other removed options:
```
			*'compatible'* *'cp'* *'nocompatible'* *'nocp'*
'compatible' 'cp'	Removed. |vim-differences| {Nvim}
```